scout/server/blueprints/variants/views.py

```diff
@@ -67,7 +67,14 @@ def variants(institute_id, case_name):
         form = FiltersForm(request.args)
         # set form variant data type the first time around
         form.variant_type.data = variant_type
-        form.chrom.data = request.args.get("chrom", None)
+        # set chromosome to all chromosomes
+        form.chrom.data = request.args.get("chrom", "")
+        # set gene panels to case-specific gene panels
```

Comment:
Hm, perhaps leave the comment out - or correct: the code implies case-specific default panels?

Reply:
😝 I know the feeling.. 
Anyway, I think the code appears sound, just saying the comment says something different. The list of panels being set is filtered with `if panel["is_default"] is True` and the comment kind of ignores that. This makes me uneasy. 😨 😉

Reply:
```suggestion
        # set gene panels to case-specific default gene panels
```